packages/teleport/src/Nodes/useNodes.ts

```diff
@@ -210,4 +209,15 @@ function makeOptions(
   });
 }
 
+// sort logins by making 'root' as the first in the list
+const sortLogins = (logins: string[]) => {
```

Comment:
i thought i was going crazy b/c i swear i just reviewed this :laughing:, we should probably put this and `makeOptions` in a shared utils file, maybe export them from `useNodes`? since they both share them :woman_shrugging: